caluma/form/format_validators.py

```diff
@@ -22,18 +21,24 @@ class BaseFormatValidator:
     ...     regex = r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
     ...     error_msg = {"en": "Not an e-mail address", "de": "Keine E-Mail adresse"}
     ```
-
     """
 
+    # `name` and `error_msg` are just dicts and get cast to `LocalizedValue` when needed.
+    #  This is to not break with the existing API.
+
     def __init__(self):
         if not all(
             [self.slug, self.regex, self.name, self.error_msg]
         ):  # pragma: no cover
             raise NotImplementedError("Missing properties!")
+        if not isinstance(self.name, dict) and isinstance(
+            self.error, dict
+        ):  # pragma: no cover
+            raise ValueError("`name` and `error_msg` must be of type `dict`!")
```

Comment:
`LocalizedValue` can handle passing it a simple string, so I'm not sure we need to enforce this. Would simplify the API a bit for monolingual projects.

```python
In [1]: from localized_fields.value import LocalizedValue
In [2]: LocalizedValue("hello world")
Out[2]: LocalizedValue<{'de': 'hello world', 'fr': None, 'it': None, 'en': None}> 0x140401073378760
```

Reply:
Done!